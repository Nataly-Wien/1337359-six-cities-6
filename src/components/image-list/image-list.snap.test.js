import React from 'react';
import {render} from '@testing-library/react';
import ImageList from './image-list';

it(`Should photos render correctly`, () => {
  const pictures = [
    `img/img1.jpg`,
    `img/img2.jpg`,
    `img/img3.jpg`,
    `img/img4.jpg`,
    `img/img5.jpg`,
    `img/img6.jpg`,
  ];

  const {container} = render(<ImageList pictures={pictures} />);

  expect(container).toMatchSnapshot();
});
