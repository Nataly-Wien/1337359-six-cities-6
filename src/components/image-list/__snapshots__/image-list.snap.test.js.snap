// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should photos render correctly 1`] = `
<div>
  <div
    class="property__gallery-container container"
  >
    <div
      class="property__gallery"
    >
      <div
        class="property__image-wrapper"
      >
        <img
          alt="Photo studio"
          class="property__image"
          src="img/img1.jpg"
        />
      </div>
      <div
        class="property__image-wrapper"
      >
        <img
          alt="Photo studio"
          class="property__image"
          src="img/img2.jpg"
        />
      </div>
      <div
        class="property__image-wrapper"
      >
        <img
          alt="Photo studio"
          class="property__image"
          src="img/img3.jpg"
        />
      </div>
      <div
        class="property__image-wrapper"
      >
        <img
          alt="Photo studio"
          class="property__image"
          src="img/img4.jpg"
        />
      </div>
      <div
        class="property__image-wrapper"
      >
        <img
          alt="Photo studio"
          class="property__image"
          src="img/img5.jpg"
        />
      </div>
      <div
        class="property__image-wrapper"
      >
        <img
          alt="Photo studio"
          class="property__image"
          src="img/img6.jpg"
        />
      </div>
    </div>
  </div>
</div>
`;
